Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Revolvers/revolvers.yml

```diff
@@ -52,6 +52,56 @@
   categories: [ HideSpawnMenu ]
   name: Customer Service Resolver
 
+- type: entity
+  parent: BaseWeaponRevolver
+  name: The Arbitrator
+  id: WeaponRevolverarbitrator
+  description: S.E.S.W.C has revamped this classic for the modern age! Civilian-grade, supports .45 mag, .35 auto, and .410 shells 
+  components:
+  - type: Sprite
+    sprite: _NF/Objects/Weapons/Guns/Revolvers/arbitrator.rsi
+    state: icon
+  - type: Clothing
+    sprite: _NF/Objects/Weapons/Guns/Revolvers/arbitrator.rsi
+    quickEquip: false
+    slots:
+    - Belt
+  - type: Item
+    size: Small
+    shape:
+    - 0,0,1,0
+    - 0,1,0,1
+  - type: Tag
+    tags:
+    - Sidearm
+  - type: Gun
+    fireRate: 2
+    availableModes:
+    - SemiAuto
+    soundGunshot:
+      path: /Audio/Weapons/Guns/Gunshots/revolver.ogg
+  - type: AmmoCounter
```

Comment:
These line can be removed as well because they are inherited from parent

Reply:
done